docs/oauth2.md

```diff
@@ -0,0 +1,30 @@
+Oauth2
+======
+
+Oauth2 is a django app that will can be used to cache your `django-oauth-toolkit`
+access token model.
+
+
+Usage
+-----
+To start caching your api access tokens, add `django_toolkit.oauth2` to your
```

Comment:
I think [the best practice](https://docs.djangoproject.com/en/1.10/ref/applications/#for-application-authors) is to put the entire path to _AppConfig_ in `INSTALLED_APPS`. If you do that, we don't need the `default_app_config`.